<ui version="4.0" >
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CMS Signer</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Identities:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyListView" name="lv_identities" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Known Peers:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyListView" name="lv_known" />
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="te_data" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Signature:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="te_sig" />
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_sign" >
        <property name="text" >
         <string>&amp;Sign</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_verify" >
        <property name="text" >
         <string>&amp;Verify</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout_CMS_Signer" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionLoad_Identity_From_File" />
    <addaction name="actionLoad_Identity_From_Storage_Device" />
    <addaction name="separator" />
    <addaction name="actionConfigure_PKCS_11_Modules" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionLoad_Identity_From_File" >
   <property name="text" >
    <string>Load Identity From &amp;File...</string>
   </property>
  </action>
  <action name="actionLoad_Identity_From_Storage_Device" >
   <property name="text" >
    <string>Load Identity From &amp;Storage Device...</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout_CMS_Signer" >
   <property name="text" >
    <string>About CMS Signer</string>
   </property>
  </action>
  <action name="actionConfigure_PKCS_11_Modules" >
   <property name="text" >
    <string>Configure &amp;PKCS#11 Modules...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyListView</class>
   <extends>QListView</extends>
   <header>mylistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cmssigner.qrc" />
 </resources>
 <connections/>
</ui>
